bench/blast/math/dense/StaticGemm.cpp

```diff
@@ -22,24 +20,27 @@ namespace blast :: benchmark
         size_t constexpr K = M;
 
         StaticMatrix<Real, M, K, columnMajor> A;
-        StaticMatrix<Real, N, K, columnMajor> B;
+        StaticMatrix<Real, K, N, columnMajor> B;
```

Comment:
Haha I guess this is why you saw some weird blips in the performance graphs :p

Reply:
I don't think so, because we benchmark for M = N = K.